pages/agent/v3/cli_env.md

```diff
@@ -1,8 +1,8 @@
 # `buildkite-agent env`
 
-The Buildkite Agent's `env` subcommands provide the ability to inspect environment variables.
+The Buildkite Agent's `env` subcommands provide the ability to inspect environment variables. In order for jobs to inspect and modify the environment variables the agent needs to have expermiment flag set with job-api like `--experiment=job-api`
```

Comment:
Now that it's below, I'll add a suggestion for removing it here:

```suggestion
The Buildkite Agent's `env` subcommands provide the ability to inspect environment variables.
```

Reply:
Yes, i should have removed that as we added it below. Done